engines/mohawk/detection_tables.h

```diff
@@ -87,6 +87,24 @@ static const MohawkGameDescription gameDescriptions[] = {
 		0,
 	},
 
+	// Myst Demo
+	// English Windows 3.11, v1.0
+	// From PC Format (UK) July, 1994
+	{
+		{
+			"myst",
+			"Demo",
+			AD_ENTRY1("DEMO.DAT", "8ff8ae264f759ea4a79cc915757f17c4"),
+			Common::EN_ANY,
+			Common::kPlatformWindows,
+			ADGF_DEMO | ADGF_NO_FLAGS,
```

Comment:
The flags here are wrong (you can't use ADGF_NO_FLAGS together with other flags). Also, it is mentioned that the movie is not playing, so this should be marked either as unstable or unsupported